src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,74 @@
+.calendar {
+  height: 100vh;
+
+  &__title {
+    display: flex;
+    justify-content: center;
+
+    padding-top: 25px;
+
+    font-size: 55px;
+    color: black;
+    cursor: pointer;
+
+    &:hover {
+      text-shadow: 3px 3px pink;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $day-gap;
+
+    max-width: $calendar-width;
```

Comment:
Better use `width` - on small screen sizes there're less than 7 rows. This task requires exact 7.

Reply:
Yes, it really is as you say